app/services/tagging_history/index_page.rb

```diff
@@ -0,0 +1,35 @@
+module TaggingHistory
+  class IndexPage
+    attr_reader :link_changes, :filter_by_user_options
+
+    def initialize(params = { link_types: ['taxons'] })
+      @filter_by_user_options = {}
```

Comment:
It may be clearer to replace this with a method that goes through the `@link_changes` and builds up the list of user options instead of doing everything in the `get_link_changes` method.

Reply:
I've now done this as part of refactoring this class in f93598b